test/unit/test/streaming/streaming.controllers.CmcdController.js

```diff
@@ -425,7 +425,7 @@ describe('CmcdController', function () {
                     return e.split('=')[0]
                 })).to.not.include('dl');
                 expect(headers[STATUS_HEADER_NAME]).to.be.undefined;
-                expect(headers[SESSION_HEADER_NAME]).to.be.empty;
+                expect(headers[SESSION_HEADER_NAME]).to.be.undefined;
```

Comment:
It's not clear why one would be an empty string, and the other undefined. In CML, when a header shard has no keys, it is not included in the resulting header object at all.